manifests/TechSmith/Camtasia/2019.0.10.yaml

```diff
@@ -0,0 +1,18 @@
+Id: TechSmith.Camtasia
+Name: Camtasia
+AppMoniker: Camtasia
+Version: 2019.0.10
+Publisher: TechSmith
+License: Copyright (C) 2020 TechSmith
+MinOSVersion: 10.0.0.0
+Homepage: https://www.techsmith.com/
+Description: Screen Capture and Recording Software
+Tags: "snagit,screen capture,recording"
+Installers: 
+  - Arch: x64
+    Url: https://download.techsmith.com/camtasiastudio/releases/2006/camtasia.exe
+    InstallerType: exe
+    Sha256: 5049d55479143da18af1fcb6d2bdd88fd051fa086a4f16f686a623e4093db6a0
```

Comment:
This is the Sha I get for version 2020.0.6. Which is not the version of this YAML file (2019.0.10)

Reply:
CC @KevinLaMS